api-gateway/src/main/resources/static/scripts/owner-details/owner-details.controller.js

```diff
@@ -8,11 +8,11 @@ angular.module('ownerDetails')
             self.owner = resp.data;
         });
 
-        $http.update('api/gateway/owners/' + $stateParams.ownerId).then(function (resp) {
+        /*$http.update('api/gateway/owners/' + $stateParams.ownerId).then(function (resp) {
```

Comment:
We should create 2 new JS file, each one containing an HTTP request like update and delete